contracts/crowdsale/distribution/FinalizableCrowdsale.sol

```diff
@@ -31,8 +31,8 @@ contract FinalizableCrowdsale is TimedCrowdsale {
      * work. Calls the contract's finalization function.
      */
     function finalize() public {
-        require(!_finalized);
-        require(hasClosed());
+        require(!_finalized, "FinalizableCrowdsale: already finalized");
+        require(hasClosed(), "FinalizableCrowdsale: not closed");
```

Comment:
` not closed` doesn't make any valuable meaning. It should be some valuable string.

Reply:
@princesinha19 
I think the reason string will be read as:
`FinalizableCrowdsale: already finalized`
`FinalizableCrowdsale: not closed`
And this will be printed on the console, thus, from my perspective, it does make sense, and no need of adding any extra text.

Reply:
@princesinha19 such a message would indicate at a glance that a transaction failed because a `FinalizableCrowdsale` is not closed. Going into the source code would then reveal that `finalize` was called in said crowdsale, causing the revert. 

What extra information would you add to help a user in such a scenario?